install-on-arch.sh

```diff
@@ -70,6 +70,8 @@ clear
 echo "We need an AUR helper. It is essential. 1) paru       2) yay"
 read -r -p "What is the AUR helper of your choice? (Default is paru): " num
 
+HELPER="paru"
```

Comment:
Since paru is the default we can just set it here, if the user doesn't select anything the value will already be set, so we're good. This will also allow the value to be scoped to the file rather than the case block